iOS Client/DevotionsController/DevotionDetailViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  DevotionDetailViewController.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 11/15/17.
+//  Copyright © 2017 Mockingbird. All rights reserved.
+//
+
+import UIKit
+import ReSwift
+
+class DevotionDetailViewController: UIViewController, StoreSubscriber {
+
+    @IBOutlet weak var monthLabel: UILabel!
+    @IBOutlet weak var dayLabel: UILabel!
+    @IBOutlet weak var verseLabel: UILabel!
+    @IBOutlet weak var verseTextLabel: UILabel!
+    @IBOutlet weak var bodyLabel: UILabel!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureBackButton()
```

Comment:
maybe we should subclass UIViewController and override viewDidLoad to configure the back button for us since we do it more than once? We could also do the store subscribe and unsubscribe in viewWillAppear and viewWillDisappear since we do that in all our VCs. Thinking out loud here, I can handle it in a later refactor if u think it's a good idea